substrate/primitives/metadata-ir/src/types.rs

```diff
@@ -170,8 +170,8 @@ pub struct ExtrinsicMetadataIR<T: Form = MetaForm> {
 	///
 	/// Note: Field used for metadata V14 only.
 	pub ty: T::Type,
-	/// Extrinsic version.
-	pub version: u8,
```

Comment:
Reminder that `extensions` below will also need to have a shape more like `Vec<(extensions_version, Vex<TransactionExtensionMetadataIR>)>` or something, but as with the other things I assume this will be a separate PR :)